src/Compilers/CSharp/Test/Syntax/Syntax/SyntaxNormalizerTests.cs

```diff
@@ -1365,5 +1362,247 @@ public void TestNormalizeScopedLocals()
+            TestNormalizeExpression("new SomeClass(){A=1,B=2,C=new SomeOtherClass(){D=5l,E=2.5f,F=new AndAnotherClass{G=7u,H=3.72m,}}}", "new SomeClass()\r\n{\r\n  A = 1,\r\n  B = 2,\r\n  C = new SomeOtherClass()\r\n  {\r\n    D = 5l,\r\n    E = 2.5f,\r\n    F = new AndAnotherClass\r\n    {\r\n      G = 7u,\r\n      H = 3.72m,\r\n    }\r\n  }\r\n}");
+            TestNormalizeExpression("new SomeClass(){A=1,B=2,C=new SomeOtherClass(){D=5l,E=2.5f,F=new AndAnotherClass{G=7u,H=3.72m,},}}", "new SomeClass()\r\n{\r\n  A = 1,\r\n  B = 2,\r\n  C = new SomeOtherClass()\r\n  {\r\n    D = 5l,\r\n    E = 2.5f,\r\n    F = new AndAnotherClass\r\n    {\r\n      G = 7u,\r\n      H = 3.72m,\r\n    },\r\n  }\r\n}");
+            TestNormalizeExpression("new SomeClass(){A=1,B=2,C=new SomeOtherClass(){D=5l,E=2.5f,F=new AndAnotherClass{G=7u,H=3.72m,},},}", "new SomeClass()\r\n{\r\n  A = 1,\r\n  B = 2,\r\n  C = new SomeOtherClass()\r\n  {\r\n    D = 5l,\r\n    E = 2.5f,\r\n    F = new AndAnotherClass\r\n    {\r\n      G = 7u,\r\n      H = 3.72m,\r\n    },\r\n  },\r\n}");
+            TestNormalizeExpression("new SomeClass(){A=1,B=2,C=new SomeOtherClass(){D=5l,E=2.5f,F=new AndAnotherClass(){}}}", "new SomeClass()\r\n{\r\n  A = 1,\r\n  B = 2,\r\n  C = new SomeOtherClass()\r\n  {\r\n    D = 5l,\r\n    E = 2.5f,\r\n    F = new AndAnotherClass()\r\n    {\r\n    }\r\n  }\r\n}");
+            TestNormalizeExpression("new SomeClass(){A=1,B=2,C=new SomeOtherClass(){D=5l,E=2.5f,F=new AndAnotherClass(){G=7u,H=3.72m}}}", "new SomeClass()\r\n{\r\n  A = 1,\r\n  B = 2,\r\n  C = new SomeOtherClass()\r\n  {\r\n    D = 5l,\r\n    E = 2.5f,\r\n    F = new AndAnotherClass()\r\n    {\r\n      G = 7u,\r\n      H = 3.72m\r\n    }\r\n  }\r\n}");
+            TestNormalizeExpression("new SomeClass(){A=1,B=2,C=new SomeOtherClass(){D=5l,E=2.5f,F=new AndAnotherClass(){G=7u,H=3.72m,}}}", "new SomeClass()\r\n{\r\n  A = 1,\r\n  B = 2,\r\n  C = new SomeOtherClass()\r\n  {\r\n    D = 5l,\r\n    E = 2.5f,\r\n    F = new AndAnotherClass()\r\n    {\r\n      G = 7u,\r\n      H = 3.72m,\r\n    }\r\n  }\r\n}");
+            TestNormalizeExpression("new SomeClass(){A=1,B=2,C=new SomeOtherClass(){D=5l,E=2.5f,F=new AndAnotherClass(){G=7u,H=3.72m,},}}", "new SomeClass()\r\n{\r\n  A = 1,\r\n  B = 2,\r\n  C = new SomeOtherClass()\r\n  {\r\n    D = 5l,\r\n    E = 2.5f,\r\n    F = new AndAnotherClass()\r\n    {\r\n      G = 7u,\r\n      H = 3.72m,\r\n    },\r\n  }\r\n}");
+            TestNormalizeExpression("new SomeClass(){A=1,B=2,C=new SomeOtherClass(){D=5l,E=2.5f,F=new AndAnotherClass(){G=7u,H=3.72m,},},}", "new SomeClass()\r\n{\r\n  A = 1,\r\n  B = 2,\r\n  C = new SomeOtherClass()\r\n  {\r\n    D = 5l,\r\n    E = 2.5f,\r\n    F = new AndAnotherClass()\r\n    {\r\n      G = 7u,\r\n      H = 3.72m,\r\n    },\r\n  },\r\n}");
```

Comment:
these are basically impossible to validate visually.  Can you change to multiline strings?

if not already there, please add tests for nesting of initializers like: 

```new C { X = { A, B, C } }` (including when A, B, C are themselves object/collection initializers.

Reply:
>  Can you change to multiline strings?

How to ensure line endings to be equal then?

> nesting of initializers

There are test for 3 nested initializers there)

Reply:
> How to ensure line endings to be equal then?

Not sure what you mean.

> There are test for 3 nested initializers there)

Can't tell.  Will be able to review once multiline.  Thanks!

Reply:
> Not sure what you mean.

I don't think (but I might me wrong) that if I have a verbatim/raw multiline string, a `\r\n` is guaranteed to be the line break and it depeneds on editor settings and OS. While at the same time, the `NormilizeWhitespace` always uses `\r\n` for line breaks. So will these tests break if they are run on, let's say, Linux or if someone formats code using `\n` as line break?

Reply:
Found `NormalizeLineEndings` method. The question is closed, gonna redo the tests

Reply:
you can always preprocess the inputs to whatever you want and specify exactly which values you want to pass into NormalizeWhitespace (including what hte newline should be).  So you can test all combinations outcomes.  e.g. `\r\n` input and output, and `\n` input and output.  These are tests, so you can do whatever you want in them.

Reply:
> There are test for 3 nested initializers there)

I looked for some (searching for `={` and `= {`) but couldn't find them.
The tests named "Nested" use `= new { ... }` which is a different thing.

Reply:
> searching for `={` and `= {`

Till this moment I didn't even know this is a thing)

> The tests named "Nested" use `= new { ... }` which is a different thing.

I didn't separate tests for initializers and anonymous object creations for simplicity reasons